pkg/services/ngalert/models/instance.go

```diff
@@ -38,8 +39,21 @@ const (
 	InstanceStateError InstanceStateType = "Error"
 )
 
-// IsValid checks that the value of InstanceStateType is a valid
-// string.
+// InstanceCauseType is an enum to differentiate between the different pending states.
+type InstanceCauseType string
+
+const (
+	// InstanceNoCause is for any state that does not come from firing or error.
+	InstanceNoCause InstanceCauseType = ""
```

Comment:
I would change this to `InstanceCauseNone` then all causes have the same prefix.

```suggestion
	InstanceCauseNone InstanceCauseType = ""
```